src/test/ui/issues/issue-75599.rs

```diff
@@ -0,0 +1,24 @@
+// run-pass
```

Comment:
```suggestion
// check-pass
```
No need to run it, it's only there for the diagnostics.

Reply:
Thanks!